app/src/frontend/building/data-components/autofill-dropdown/autofill-dropdown.tsx

```diff
@@ -60,15 +60,14 @@ export const AutofillDropdown: React.FC<AutofillDropdownProps> =  props => {
             {
                 options.map(option =>
                     <div
-                        key={option.id}
                         onMouseDown={e => /* prevent input blur */ e.preventDefault()}
                         onClick={e => {
                             props.onSelect(option.value);
                             // close dropdown manually rather than through input blur
                             props.onClose();
                         }}
                     >
-                        {option.value} ({option.id})
```

Comment:
@mz8i So, another option would be to query as so far - but simply to stop showing it to user, right?